HoneyBadgers.WebApp/Views/Shared/_MoviePartialView.cshtml

```diff
@@ -0,0 +1,46 @@
+﻿@model IEnumerable<HoneyBadgers.Logic.Movie>
+
+@foreach (var movie in Model)
+{
+    <div id="movie-partial-view" class="hb-movie card mb-3" style="max-width: 600px;" onclick="">
+        <div class="row g-0">
+            <div class="col-md-4">
+                <img src="@movie.Poster" class="img-fluid rounded-start" alt="@movie.Title">
+            </div>
+            <div class="col-md-8">
+                <div class="card-body">
+                    <div class="row hb-movie__header">
+                        <div class="col-md-10">
+                            <h6 class="card-title">@movie.Title</h6>
+                        </div>
+                        <div class="col-md-2">
+                            <i class="far fa-heart"></i>
```

Comment:
`<i>` ? co to jest?

Reply:
w html służy to pisania tekstu w _italic_ ale w tym wypadku jest wykorzystywane do wyświetlenia ikonki, dodałam nam bibliotekę dla ikonek - font-awesome. Mogę wrzucić link na slacka

Reply:
Yhym :D to nie wiedziałem, myślałem że italic się przez właściwości/font w <p> zmienia :3